app/components/reward.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class RewardComponent extends Component {
+    @tracked rewardPoints = '10$';
+    @tracked quizPoints = '546';
+    @tracked trainingPoints = '456';
```

Comment:
try to have different numbers for quiz and training

Reply:
done